Resources/Catalog/FileCatalog.py

```diff
@@ -46,8 +46,10 @@ def __init__( self, catalogs = [], vo = None ):
     self.vo = vo if vo else getVOfromProxyGroup().get( 'Value', None )
 
     self.opHelper = Operations( vo = self.vo )
-    if type( catalogs ) in types.StringTypes:
-      catalogs = [catalogs]
+    if catalogs == None:
```

Comment:
What do you mean? I changed the default value to None as otherwise pylint creates an error at the line catalogs = catalogs.split(',')
and also because it is not good practice and can indeed be quite dangerous to set a default value to []